Gemfile

```diff
@@ -38,8 +38,8 @@ gem 'momentjs-rails', '>= 2.9.0' # needed for human-friendly textual dates
 gem 'font-awesome-rails'
 
 # needed to run graders in the background
-gem 'delayed_job', github: 'dsander/delayed_job', branch: 'rails51' # FIXME: github
```

Comment:
Ok.  Is it correct to say that the *existing* {JUnit, JavaStyle, Checker, RacketStyle}Graders will all continue to work as they currently are, with this change?  Or do all of those need to be rewritten concurrently with this change, to use Steno as the backend?  I'm *really* hesitant to do that right now -- I'd rather you beta-test Steno with your courses this fall (where you can babysit it and keep track of any necessary changes) than with Fundies (where I probably won't have any bandwidth to do the same).